ols/app/metrics/metrics.py

```diff
@@ -28,5 +41,15 @@
 selected_provider = Info("selected_provider", "Selected provider")
 selected_model = Info("selected_model", "Selected model")
 
-# register metric
-metrics_app = make_asgi_app()
+
+@router.get("/metrics/")
```

Comment:
`/metrics` is better, this is no child path to this.

Reply:
indeed, i also agree but i left the path as it was before the change 
if needed i can change the existing tests that rely on "/metrics/"

Reply:
let's keep it for /metrics/ for now, and make another PR to change path here as well as in the tests. 
It seems not many references to take care of :) 
```shell
➜  lightspeed-service git:(main) ✗ ag '/metrics/'
ols/app/main.py
62:    if not path.endswith("/metrics/"):

README.md
223:curl 'http://127.0.0.1:8080/metrics/'

tests/integration/test_metrics.py
24:    response = client.get("/metrics/")
26:    # check that the /metrics/ endpoint is correct and we got
116:    assert 'response_duration_seconds_count{path="/metrics/"}' in response_text
117:    assert 'response_duration_seconds_sum{path="/metrics/"}' in response_text

tests/e2e/test_api.py
29:    response = client.get("/metrics/", timeout=BASIC_ENDPOINTS_TIMEOUT)
31:    # check that the /metrics/ endpoint is correct and we got
544:    response = client.get("/metrics/", timeout=BASIC_ENDPOINTS_TIMEOUT)
563:    assert 'response_duration_seconds_count{path="/metrics/"}' in response.text
564:    assert 'response_duration_seconds_sum{path="/metrics/"}' in response.text
```